### ISSUE TITLE
Why tab inactive content does`t unloaded?

### ISSUE THREAD

**Issue by emejibka** [ButchersBoy/Dragablz#138]
Test case;
1. Create drazablz control, bind itemssource to viewmodel collection
2. Create dataTemplate for type
3. Run app. Inactive tab content does not unloaded when I switch tab. TabControl from wpf unload components from inactive tab.

**Comment by ButchersBoy** [comment]
Yes, currently by design, as this can drastically slow things down as layout/measuring rendering processes kick in.

**Comment by Mhyrtas** [comment]
I actually doubt it is still true though. I'm using 2 different controls (TabControl with a non reloading feature and TabablzControl). TabablzControl is much more slower cause it unloads my viewmodels. I tried to plug the non reloading class in it but it wont work at all.
